Tasks/GoTool/gotool.ts

```diff
@@ -16,8 +15,8 @@ try {
 
 async function run() {
     try {
-        let versionSpec = tl.getInput('version', true);
-        await getGo(versionSpec);
+        let version = tl.getInput('version', true).trim();
```

Comment:
Use .replace(/ /g,'') instead of trim(). I think motive here is to remove all the white spaces!

Reply:
trim should be fine. As striping leading and trailing space make sense, but removing in between spaces not required.
We can only control user action to certain limit.